test/controllers/tickets_controller_test.rb

```diff
@@ -78,6 +78,7 @@ def setup
     assert user.tickets.all.first.approved?
 
     # Deletion
+    ticket_details["link"] = "http://link2.to"
```

Comment:
`ticket_details[:link]` would look better. Making the change myself and merging.